feature-group-definitions/constraint-validation.yml

```
@@ -1,4 +1,5 @@
 name: Constraint validation API
+description: The Constraint Validation API provides a programmatic way to validate form controls.
```

I think this is just around the edge of how much is reasonable to spell out, but I think it works:

```suggestion
description: The `checkValidity()`, `reportValidity()` and `setCustomValidity()` methods are a programmatic way to validate form controls.
```